packages/web/src/components/track/DownloadSection.tsx

```diff
@@ -179,6 +186,18 @@ export const DownloadSection = ({ trackId }: DownloadSectionProps) => {
     [trackId, stemTracks]
   )
 
+  const renderDownloadAllButton = () => {
+    if (shouldHideDownload || !isDownloadAllTrackFilesEnabled) {
```

Comment:
can we just get rid of this feature flag yet?

Reply:
probably lol